### DISCUSSION TITLE
Debug mode off not working....

### DISCUSSION THREAD

**Discussion by rica-carv** [e107inc/e107#5328] [Q&A Core] [answered]
I've tryed to use the debug mode via url, but when trying to setting it off, my site has gonne beserk....
![image](https://github.com/user-attachments/assets/15da5e3f-67f0-41e1-b2a2-8dacc53a65c7)

Not even admin area is safe anymore....
![image](https://github.com/user-attachments/assets/59801d7e-8f63-41e8-b9ff-31a3cf3f222d)

Anyone has ideas how to solve it?
I can't even have access to the menu area in admin area!!!!

**Comment by PH5HP** [comment]
Hi,

I think the option debug mode on or off would be somewhere in the E107 database  which your site uses. So I would look there with PHPmyAdmin and search in the database and if found turn it off.

Kind regards,
Henk - https://ph5hp.nl

**Comment by Jimmi08** [comment]
Just close the browser ...  and clear cache...  be sure that you whilelisted your IP in ban part. 

Something was changes... my browser app is not working anymore either...

![image](https://github.com/user-attachments/assets/373d0b78-47d3-4e29-adba-fed3796bc7b7)

**Reply by rica-carv** [reply]
@Jimmi08 Already tryed everything, even shut down server and fresh restart. E107 just gonne beserk. Debug mode is off, but it still debugs.........
![image](https://github.com/user-attachments/assets/93556559-9302-4a96-a609-8cefb7df6f90)

**Comment by rica-carv** [comment, marked as answer]
Solved by reinstalling e107.... weird behaviour....
